src/components/molecules/Header.tsx

```diff
@@ -29,8 +29,8 @@ const Header: FC = () => {
   }, [store]);
   //login or logout- depend on authentication state
   let authElement;
-  if (store.isUserAuthenticated) {
-    authElement = <UserProfileHeader name={store.userName} />;
+  if (store.userInfo.firstName) {
+    authElement = <UserProfileHeader firstName={store.userInfo.email} />;
```

Comment:
(store?.userInfo?.firstName)
`firstName={store.userInfo.firstName}`?

Reply:
user info is passed directly now